client/package.json

```diff
@@ -34,6 +36,8 @@
 		"web-vitals": "^2.1.4"
 	},
 	"scripts": {
+		"predeploy": "npm run build",
```

Comment:
I think "npm run build" runs "react-scripts build" anyway, so might be better to just skip to that with this command?

Reply:
It should be there. When you run `npm run deploy`, it execute `npm run build`. Then because `npm run build` is executed, it run the command `reacts-script build` as we normally use `npm run build`